x/multi-staking/keeper/store_test.go

```diff
@@ -86,3 +89,43 @@ func (suite *KeeperTestSuite) TestSetValidatorMultiStakingCoin() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestSetMultiStakingLock() {
+	suite.SetupTest()
+	delAddr := testutil.GenAddress()
+	valAddr := testutil.GenValAddress()
+
+	lock := types.MultiStakingLock{
+		LockID: types.LockID{
+			MultiStakerAddr: delAddr.String(),
+			ValAddr:         valAddr.String(),
+		},
+		LockedCoin: types.MultiStakingCoin{
+			Denom:      gasDenom,
+			Amount:     sdk.NewIntFromUint64(1000000),
+			BondWeight: sdk.NewDec(1),
+		},
+	}
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expError bool
+	}{
```

Comment:
there's no fail case for this test

Reply:
This is different test for check the store, not related in invariants. We should continue this test in different PR

Reply:
there should be fail case anyway because there is tc.expError.

Reply:
If so I think we should move it to a separate PR

Reply:
it's okay to put the test in this pr